### ISSUE TITLE
Duplicate Strings in the extracted text from PDF

### ISSUE THREAD

**Issue by kannanbas** [pymupdf/PyMuPDF#379]
_**Please provide all mandatory information!**_

## Describe the bug (mandatory)
When i extract text from pdf file i am getting the duplicates text of two or more times.

## To Reproduce (mandatory)
```
with fitz.open(fp) as doc:
     text = ''
     for n in range(0,doc.pageCount):
           page = doc[n]
           text = text+page.getText("text")
```
when i tried to run the above code and extract the text from pdf i am getting exact text two to three times

## Expected behavior (optional)
I need to extract the text without the duplicate string
## Your configuration (mandatory)
 - Operating system - Windows 10
 - Python version - Python 3.7.4
 - PyMuPDF version - Pymupdf 1.14.19 PIP Installation
For example, the output of `print(sys.version, "\n", sys.platform, "\n", fitz.__doc__)` would be sufficient (for the first two bullets).
Sample example:
what i am getting:
DEMO - Sliced Invoices Suite 5A-1204 123 Somewhere Street Your City AZ 12345 admin@slicedinvoices.com DEMO - Sliced Invoices Suite 5A-1204 123 Somewhere Street Your City AZ 12345 admin@slicedinvoices.com
What i need is:
DEMO - Sliced Invoices Suite 5A-1204 123 Somewhere Street Your City AZ 12345 admin@slicedinvoices.com

**Comment by JorjMcKie** [comment]
This is a known issue of MuPDF and has been reported as issue already in #218 some time ago.
MuPDF **_should_** have corrected this in a later version, so you may be successfull when migrating to v1.16.3.

**Comment by JorjMcKie** [comment]
I just checked it out for an example PDF: the issue should be solved with version 1.16.x
